pkg/apis/wildfly/v1alpha1/wildflyserver_conversion.go

```diff
@@ -0,0 +1,129 @@
+package v1alpha1
```

Comment:
Does the conversion has to be in the "old" version?
It would feel more natural that it goes into the "new" version( that knows the old package).

Reply:
We convert from spoke to hub and from hub to another spoke. In our currently implementation, v1alpha2 is our Hub (the stored version in kubernetes) and v1Alpha1 is our spoke, currently there is only one spoke. This should clear up these concepts: https://book.kubebuilder.io/multiversion-tutorial/conversion-concepts.html

We have this conversion file here to implement the conversion, because we need to convert from v1alpha1, which implements the Convertible interface, to the type that implements Hub, in out case v1alpha2. Since we choose to store in Kubernetes the v1alpha2, we need to make v1alpha1 implement Convertible interface and then, add conversion methods here in the "old" version.